### PR TITLE
Update dev dependencies

### PR DESCRIPTION
**Pull request by jander99**
- Updated lingui in every class module and main package.json. 
- Updated prettier, fixed one prettier warning 
- Lingui types have been deprecated
- Bump @types/react-virtualized

### CODE REVIEW

**Comment by emallson** [comment]
@jander99 can you rebase this against the shadowlands branch? looks like some merged pull requests are mixed in

File: analysis/magefire/src/CHANGELOG.tsx
Comment (Pewtro) [inline analysis/magefire/src/CHANGELOG.tsx]: Seems like a bunch of fire changes are in this - probably related to a PR that has been merged, so try and merge shadowlands branch into this PR so we're sure what's being added by this PR

**Review by Pewtro — COMMENTED** [review]
Once you merge the shadowlands branch back into this, I think it'll be fine. 

Are we getting access to new features from this bump, or was it merely a "might as well do it"? 
I see we're getting lingui types directly from the lib which is nice.

**Comment by jander99** [comment]
@emallson Pulled fresh upstream and merged it in. Looks to resolve the weird network graph of out-of-sync commits. 

@Pewtro These are mostly dev dependencies and bumping things that don't completely break anything including rendering tests. I'm getting my feet wet with node/react as it's been a fairly long while since working on this type of project (Java guy) so figured I would take a shot at getting libraries updated and working on a bunch of Sonar findings.

**Review by Pewtro — APPROVED** [review]
LGTM - if you want to you can update the changelog to include a more accurate date of when the change would be merged in

**Pull request merged**
